client/src/document/prism-overrides.scss

```diff
@@ -0,0 +1,12 @@
+// NOTE! There are more overrides in Kuma
+// See styles/components/syntax/_base.scss
+// It's not clear if they're needed or still relevant.
```

Comment:
Perhaps that explains why this: http://localhost:3000/en-US/docs/Web/API/VRFrameData/timestamp looks so different from https://developer.mozilla.org/en-US/docs/Web/API/VRFrameData/timestamp
On the Yari page it looks like the background is a very different shade of gray.

Reply:
Actually @schalkneethling I think the only thing you need to do is to stop with this `<pre><code>...` nonsense and then you get the right background color. 
See the screenshot in: https://github.com/mdn/yari/issues/1072